### ISSUE TITLE
Just send plain text mail

### ISSUE THREAD

**Issue by SoltauFintel** [bbottema/simple-java-mail#133]
Great lib! But how can I send a just plain text mail? My mail has no attachments and no HTML part.

If I use
```
		mailer.sendMail(EmailBuilder.startingBlank()
				.to(to)
				.from(fromName, fromMailAddress)
				.withSubject(subject)
				.withPlainText(plainText)
				.clearHTMLText()
				.buildEmail());

```

the received mail contains this:

```
Content-Type: multipart/mixed; 
	boundary="----=_Part_0_1023714065.1521359548014"
(...)
------=_Part_0_1023714065.1521359548014
Content-Type: multipart/related; 
	boundary="----=_Part_1_1599771323.1521359548027"

------=_Part_1_1599771323.1521359548027
Content-Type: multipart/alternative; 
	boundary="----=_Part_2_1876631416.1521359548027"

------=_Part_2_1876631416.1521359548027
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

Dies ist ein Test.
=C3=84=C3=96=C3=9C=C3=9F
2018-03-18T08:52:27.960
------=_Part_2_1876631416.1521359548027--

------=_Part_1_1599771323.1521359548027--

------=_Part_0_1023714065.1521359548014--

```

As far as I understand SMTP it should be possible to send just plain text without the use of MIME parts (if there are no attachments and no HTML part). Background: My Thunderbird mail app thinks at the first second the mail contains an attachment. It shows the paperclip icon for a second. When I open the mail it shows it very well.

I use this jitpack version: `compile 'com.github.bbottema:simple-java-mail:5.0.1'`

**Comment by bbottema** [comment]
Hi @SoltauFintel, the way Simple Java Mail works is by utilizing a standardized structure that supposedly always work with any client. Today it is expected that a client will successfully parse this standard structure, which is simple in terms of MimeParts and related RFC's.

Unfortunately, by the sound of it Thunderbird is a little behind times by about ten years! I check Thunderbird myself as well and consider using the even simpler setup for just text mails.

**Comment by bbottema** [comment]
Using the even simpler method for just text mails just became a little more complex, because I just added iCalendar support (#138). This can be combined with either text or HTML body.

Fixing this issue with Thunderbird will solve it for text mails, but not for text mails which also contain a Calendar event (but not HTML text). I'm not sure if there is a fitting solution here...

**Comment by bbottema** [comment]
@SoltauFintel I just tried this in Thunderbird, but it works fine for me. Are you using an older Thunderbird version  by any chance?

**Comment by bbottema** [comment]
I found that just the mere presence of a MimeMultipart "mixed" (with attachments or not) makes Outlook add the attachment icon in the mail list.

**Comment by bbottema** [comment]
Closing in favor of #144.

**Comment by SoltauFintel** [comment]
@bbottema  Thanks for the fix in 6.0.5 :-)
